src/coreclr/vm/arm64/CachedInterfaceDispatchCoreCLR.asm

```diff
@@ -0,0 +1,83 @@
+; Licensed to the .NET Foundation under one or more agreements.
+; The .NET Foundation licenses this file to you under the MIT license.
+
+#include "ksarm64.h"
+#include "asmconstants.h"
+#include "asmmacros.h"
+
+#ifdef FEATURE_CACHED_INTERFACE_DISPATCH
```

Comment:
This is done in both the virtual and cached interface cases. Would it be better to just not include these files via the CMakeLists.txt? Seems there is no need to include the file in compilation if it is completely ifdef'd away.

Reply:
Doing that and having the support for this being conditional on chk/debug/release is actually ... fairly messy, so I moved to using #ifdefs. This will make this change avoid being a blocker for future movement to use the ninja multiconfig generator on Linux instead of our current cmake generator.